### PR TITLE
Setup URQL with own Backend and implement basic Player List + GraphQL Codegen

### LINKED ISSUE #53: Setup URQL with own backend

### PR DESCRIPTION
**Pull request by hammadj**
closes #53

### CODE REVIEW

File: packages/web/components/Container.tsx [hunk 1]
```diff
@@ -1,29 +1,24 @@
...
+export const PageContainer: React.FC<Props> = ({ children, ...props }) => (
   <Flex
     bgSize="cover"
-    w="100vw"
+    w="100%"
     h="100vh"
     p={12}
     direction="column"
     align="center"
+    backgroundSize="cover"
```
Comment (dan13ram) [inline packages/web/components/Container.tsx:14]: there's bgSize cover property set already

File: packages/web/pages/login.tsx [hunk 2]
```diff
@@ -11,7 +11,10 @@ import MetaGameImage from '../public/images/metagame.png';
 const Login: React.FC = () => {
   const [step, setStep] = useState(0);
   return (
-    <PageContainer backgroundImage={`url(${BackgroundImage})`}>
+    <PageContainer
+      backgroundImage={`url(${BackgroundImage})`}
+      backgroundSize="cover"
```
Comment (dan13ram) [inline packages/web/pages/login.tsx:16]: Isn't `backgroundSize` redundant here?

File: packages/design-system/src/theme.ts [hunk 3]
```diff
@@ -53,8 +59,14 @@ export const theme: Theme = {
...
       900: '#07021d',
     },
+    diamond: '#40e8ec',
+    platinum: '#81b6e3',
+    gold: '#d0a757',
+    silver: '#b0b0b0',
+    bronze: '#a97142',
     offwhite: '#F6F8F9',
     blue02: 'rgba(79, 105, 205, 0.2)',
+    dark60: 'rgba(0,0,0, 0.6)',
```
Comment (dan13ram) [inline packages/design-system/src/theme.ts:69]: we should come up with a more readable and uniform naming convention for translucent colors.
black60 sounds better to me than dark60. 
Maybe I should change blue02 to blue20 as well.

Comment (hammadj) [reply packages/design-system/src/theme.ts:69]: Yea, want to consolidate the colours a lot more. e.g. purpleBoxDark should just be using purple.800, purple tag should be purple.400, etc. Will update blue02 to 20 though.

**Pull request merged**